### ISSUE TITLE
Get inspired

### ISSUE THREAD

**Issue by StefanCristian** [bogdanseczkowski/OPTINUX#8]
https://gitlab.com/argent

argent-ws is the overlay, together with own eclass of building kernel automatically
argentws-build contains all the /etc/portage settings
argentws-skel / argentws-live are the /etc/skel and live scripts that boot the system

iso.argentlinux.io - the ISOs with full systemd + dracut booting
pkgwork.argentlinux.io - 19GB gentoo precompiled packages including the kernel linux-argent / argent-sources ebuilds
full-working LXQT + a lot of IDEs and stuff

**Comment by bogdanseczkowski** [comment]
I don't want to use precompiled packages because of the whole optimization thing, and that prebuilt packages are a potential security hazard. But I'll have a look

**Comment by StefanCristian** [comment]
@bogdanseczkowski yeah, the precompiled packages always are and always will be a potential security hazard, but that doesn't mean you can't have a automated way to download, install and recompile them thereafter :)

**Comment by Kreyren** [comment]
@StefanCristian You know how lazy am i to read anything.. just be emergefull and tell me if there is anything i could use in [KSEGIS](https://github.com/Kreyren/KSEGIS/).. thanks :3 <3
